lepton/azure.go

```diff
@@ -106,20 +107,26 @@ func (a *Azure) GetResourceManagementAuthorizer() (autorest.Authorizer, error) {
 	return armAuthorizer, err
 }
 
-func (a *Azure) getImagesClient() compute.ImagesClient {
+func (a *Azure) getImagesClient() (*compute.ImagesClient, error) {
 	vmClient := compute.NewImagesClientWithBaseURI(compute.DefaultBaseURI, a.subID)
-	authr, _ := a.GetResourceManagementAuthorizer()
+	authr, err := a.GetResourceManagementAuthorizer()
```

Comment:
the resource management authorizer is required in every azure client (images, vms, virtual networks, etc). 

The handling of errors every time we get a new client adds boilerplate and in some situations unnecessary code repetitions.

Reply:
errors need to be handled regardless but if it's used by a common set of methods could potentially abstract it out

Reply:
https://github.com/nanovms/ops/issues/778